pages/_app.tsx

```diff
@@ -46,6 +47,12 @@ function MyApp({ Component, pageProps }: AppProps): JSX.Element {
                     <main className="container mt-4 md:mt-0 justify-self-center overflow-auto">
                         <Component {...pageProps} />
                     </main>
+                    {!!pageProps.toc?.length && (
+                        <div className="hidden lg:block w-60 px-3 overflow-auto">
+                            <h3>Table of Content</h3>
+                            <TableOfContent titles={pageProps.toc} />
+                        </div>
+                    )}
```

Comment:
We can't have this here because they could be some pages that don't have the TOC.

Reply:
That's why we have this validation: `!!pageProps.toc?.length`. Should I move this to UrlTitle?

Reply:
I think so.